rhoas-cli/README.adoc

```diff
@@ -151,30 +127,43 @@ $ rhoas --version
 
 .Procedure
 
-. Review the link:https://github.com/redhat-developer/app-services-cli/blob/main/scripts/install.sh[`install.sh` installation script^].
+. Navigate to the https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/overview[Red Hat OpenShift Streams for Apache Kafka] page on the Red Hat Developer Portal.
+
+. Click https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/download[*Install/Specifics*] and navigate to the bottom of the page to the *All Downloads* section.
+
+. Click the *Download* button from the macOS row in the *RHOAS CLI* downloads table to download the `rhoas` zip file.
 
-. Run the `install.sh` script:
+. Unzip the file to extract the `rhoas` binary: 
 +
 --
 [source,shell]
 ----
-$ curl -o- https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/scripts/install.sh | bash
+$ unzip rhoas-${version}-darwin.zip
```

Comment:
Same comment as the previous section about showing only the command without the prompt or output.